docs/en/siem/installation.asciidoc

```diff
@@ -30,13 +32,77 @@ for sorting. For more information, see {kib}.
 [[install-beats]]
 === Install {beats} shippers
 
-To populate the security UI with metrics and
-log data, you need to install and configure the following shippers:
+To populate the {siem-ui} with hosts and network data, you need to install and
+configure Beats on the endpoints you want to collect and send data:
 
-* https://www.elastic.co/products/beats/packetbeat[{packetbeat}] for analyzing
-network packets 
 * https://www.elastic.co/products/beats/filebeat[{filebeat}] for forwarding and
 centralizing logs and files
 * https://www.elastic.co/products/beats/auditbeat[{auditbeat}] for monitoring
 directories for file changes
+* https://www.elastic.co/products/beats/winlogbeat[{winlogbeat}] for centralizing logs
+* https://www.elastic.co/products/beats/packetbeat[{packetbeat}] for analyzing
+network packets 
+
+NOTE: {siem-soln} also works with custom and third-party data sources in addition to Beats.
+{ecs-ref}[Elastic Common Schema (ECS)] makes this possible. Any data tagged as `host.name`
+can be used in {siem-soln}. 
+
+You can install {beats} from {kib} or from the command line.
+
+[float]
+==== Install {beats} from {kib}
+
+Follow the instructions in the Add Data section of the {kib} home page. Click
+*Add log data* or *Add metrics*, and follow the links for the types of data you
+want to collect.
+
+[role="screenshot"]
+image::add-data.png[]
+
+[float]
+==== Download and install {beats} from the command line
+
+If your data source isn't in the list, or you want to install {beats} the old
+fashioned way:
+
+* *{filebeat} and {filebeat} modules.* See the
+{filebeat-ref}/filebeat-modules-quickstart.html[{filebeat} modules quick start]
+and enable modules for the events you want to collect. If there is no module
+for the events you want to collect, see the
+{filebeat-ref}/filebeat-getting-started.html[{filebeat} getting started] to
+learn how to configure inputs.
+
+* *Auditbeat.* See {auditbeat-ref}/auditbeat-getting-started.html[{auditbeat} getting started].
+
+* *Winlogbeat.* See {winlogbeat-ref}/winlogbeat-getting-started.html[{winlogbeat} getting started].
+
+* *Packetbeat.* See {packetbeat-ref}/packetbeat-getting-started.html[{packetbeat} getting started].
+
+[float]
+=== Enable modules and configuration options
+
+For either approach, you need to enable modules in {auditbeat} and {filebeat}
+to populate the {SIEM-ui} with data.
+
+To populate *Hosts* data, enable these {auditbeat} modules:
+
+* {auditbeat-ref}/auditbeat-module-system.html[System module  - Linux, macOS, Win]
+* {auditbeat-ref}/auditbeat-module-auditd.html[Auditd module (Linux Kernel Audit info)]
+* {auditbeat-ref}/auditbeat-module-file_integrity.html[File integrity module (FIM) - Linux, macOS, Win]
+
+
+To populate *Network* data, enable these {filebeat} modules:
```

Comment:
We should list Packetbeat first here, with a mention of what you need to configure in Pb, to populate the SIEM app (flows & DNS at the very least).

The phrasing of enabling these Fb modules could be adjusted as well. Users need to pick & choose the modules that make sense for them, not all of them are required. A lot of them actually overlap in functionality.

Reply:
@webmat - let's work together on this